scenarios/openshift/detach_ebs_volumes.yaml

```diff
@@ -0,0 +1,9 @@
+aws_access_key_id:                            # only if system-wide credentials are not
```

Comment:
We might want to add a reference around setting up aws secrets similar to node scenarios: https://github.com/krkn-chaos/krkn/blob/main/docs/node_scenarios.md#aws. This way they do not get exposed in the stdout. Thoughts?

Reply:
agree on this, we currently configure aws connection just based on enviornment variables we should see if we can do the same set up here

Reply:
...and maybe drop the `ebs` from the name to make it CSP-agnostic?

Reply:
If the aws cli is configured it should take those credentials if other are not specified so I will put the reference how to setup the credentials in documentation and will remove the option to put there different credentials